praktikumsplaner-frontend/frontend/src/directives/Security.ts

```diff
@@ -0,0 +1,82 @@
+import Vue, { watch } from "vue";
+import Vuetify from "vuetify/lib";
+import { useUserStore } from "@/stores/user";
+import { APP_SECURITY } from "@/Constants";
+
+Vue.use(Vuetify);
+
+Vue.directive("security-allow", {
```

Comment:
both directive look very similar. I think using an expression or modifier ist the better way instead of have the same code twice.